### PR TITLE
Get ready for MenuBundle 2.0

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by WouterJ** [comment]
Ready to merge

File: composer.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
         "symfony/framework-bundle": "~2.3",
         "symfony-cmf/core-bundle": "~1.1",
         "symfony-cmf/routing-bundle": "~1.2",
-        "symfony-cmf/menu-bundle": "~1.2",
-        "symfony-cmf/content-bundle": "~1.0",
+        "symfony-cmf/menu-bundle": "2.0.*",
+        "symfony-cmf/content-bundle": "dev-menu_20",
```
Comment (dbu) [inline composer.json]: do we want to merge with this? if menu 2.0 is ready, should it become master?

**Comment by dbu** [comment]
the pr title still has a `WIP`. 
do you want to squash the commits?

**Comment by WouterJ** [comment]
Merged the PR on ContentBundle (it supports both versions now) and squashed the commits. Again, ready to merge

**Comment by dbu** [comment]
cool!

**Pull request merged**